hoomd/md/ForceComposite.cc

```diff
@@ -973,6 +986,42 @@ void ForceComposite::updateCompositeParticles(uint64_t timestep)
         updated_pos += dr_space;
         quat<Scalar> updated_orientation = orientation * local_orientation;
 
+        // calculate the tangential velocity of constituents
+        const Scalar4 vel_mass = h_velocity.data[particle_index];
+        double mass = vel_mass.w;
+        quat<Scalar> angvel_body = conj(orientation) * angmom * 0.5;
+        vec3<Scalar> av_body = angvel_body.v;
```

Comment:
Wrap constants in `Scalar`, and try to use clear variable names.
```suggestion
        quat<Scalar> angvel_body_quat = Scalar(0.5) * conj(orientation) * angmom;
        vec3<Scalar> angvel_body = angvel_body_quat.v;
```